localstack/services/cloudformation/models/iam.py

```diff
@@ -164,6 +166,46 @@ def _pre_delete(resource_id, resources, resource_type, func, stack_name):
         }
 
 
+class IAMAccessKey(GenericBaseModel):
+    @staticmethod
+    def cloudformation_type():
+        return "AWS::IAM::AccessKey"
+
+    def get_physical_resource_id(self, attribute=None, **kwargs):
+        return self.props.get("UserName")
+
+    def fetch_state(self, stack_name, resources):
+        user_name = self.resolve_refs_recursively(stack_name, self.props.get("UserName"), resources)
+        keys = aws_stack.connect_to_service("iam").list_access_keys(UserName=user_name)[
+            "AccessKeyMetadata"
+        ]
+
+        return keys[-1]
+
+    @staticmethod
+    def get_deploy_templates():
+        def _delete(resource_id, resources, resource_type, func, stack_name):
+            iam_client = aws_stack.connect_to_service("iam")
+            resource = resources[resource_id]
+            props = resource["Properties"]
+            user_name = props["UserName"]
+            access_key_id = props["AccessKeyId"]
+
+            try:
+                iam_client.delete_access_key(UserName=user_name, AccessKeyId=access_key_id)
+            except ClientError as err:
+                if "NotSuchEntity" not in err.response["Error"]["Code"]:
+                    raise
+
+        return {
+            "create": {
+                "function": "create_access_key",
+                "parameters": ["UserName", "Serial", "Status"],
+            },
+            "delete": {"function": _delete},
```

Comment:
```suggestion
            "delete": {"function": _delete},
```

Any reason for not just mapping the parameter names here? (i.e. basicallly mapping `UserName=user_name, AccessKeyId=access_key_id`)

Reply:
@dominikschubert I based the code on the other models, I'm not sure how I would use the mapped parameters in the `_delete` function.